server/server.go

```diff
@@ -60,6 +69,7 @@ func setupRouter(logger *zap.Logger, db *sql.DB) *gin.Engine {
 	example.GET("/error", env.errorHandler)
 	example.GET("/panic", env.panicHandler)
 	example.GET("/db-check", env.dbCheckHandler)
+	example.GET("/private", middleware.Auth(authenticator), env.privateHandler)
```

Comment:
I think it should be a part of `auth` or `user` group, not a part of the example. But it's okay if you are going to do it in next request